### ISSUE TITLE
Need For Clarification  For Success Criteria In WCAG2Mobile

### ISSUE THREAD

**Issue by utkudyn-a11y** [w3c/matf#272]
- 1.4.10 – Reflow: There is no way to zoom %400 for mobile apps except for mobile devices zoom feature in accessibility setting. But zoom feature in accessibility settings works exactly in opposite way in their nature. Should we use an accessibility overlay to pass this criteria?
- 1.4.12 – Text Spacing: There is no functionality or extension (as I konw) in Android and IOS to adjust text spacing. This is not a practical criteria for mobile. Again, should we use an accessibility overlay to pass this criteria?
- 2.1.4 and 2.4.7 should be tested by using an external keyboard I guess, right?

I think there should be more clarification for testing methodology for mobile.
